BankManagerConsoleApp/BankManagerConsoleApp/Bank.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  Bank.swift
+//  BankManagerConsoleApp
+//
+//  Created by steven on 2021/04/28.
+//
+
+import Foundation
+
+struct BankPrinter {
+
+    static func printMenu() {
+        print("1 : 은행개점")
+        print("2 : 종료")
+        print("입력 : ", terminator:"")
+    }
+    
+    static func printFinishPharse(_ totalCustomerNumber: Int, _ totalSecond: Double) {
+        print("업무가 마감되었습니다. 오늘 업무를 처리한 고객은 총 \(totalCustomerNumber)명이며, 총 업무시간은 \(totalSecond.cutNumberToTwoDecimalPoint())초입니다.")
+    }
+    
+    static func printWrongInputPharse() {
+        print("잘못된 입력입니다. 다시 입력해주세요.")
+    }
+
+}
+
+extension Double {
+
+    func cutNumberToTwoDecimalPoint() -> String {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.roundingMode = .floor
+        numberFormatter.maximumFractionDigits = 2
+        guard let result = numberFormatter.string(for: self) else {
+            return String(self)
+        }
+        return result
+    }
+
+}
+
+struct InputHelper {
+    
+    static func receive() -> String? {
+        guard let input = readLine() else {
+            return nil
+        }
```

Comment:
guard문은 길어서 개행이 필요하지 않으면 다음과 같이 쓰는게 더 깔끔하고 보기 좋습니다!
```suggestion
        guard let input = readLine() else { return nil }
```

Reply:
이 부분도 한줄로 처리해주었어요 ㅎㅎ